i18n/locale-en.json

```diff
@@ -0,0 +1,265 @@
+{
+  "I18N_GALLERY_PAGE_TITLE": "Oppia - Gallery",
```

Comment:
Not directly related to this file, but have you thought about RTL (Right to Left) language support when adding i18n support? Possible useful information can be found at https://www.mediawiki.org/wiki/Directionality_support.